_data/simple-icons.json

```diff
@@ -8010,7 +8014,7 @@
         {
             "title": "Scrum Alliance",
             "hex": "009FDA",
-            "source": "https://www.scrumalliance.org/ScrumRedesignDEVSite/media/ScrumAllianceMedia/Files%20and%20PDFs/Infographics/S_BrandGuidelines_2018_rev.pdf",
+            "source": "https://scrum.mx/advanced-certified-scrum-master",
             "guidelines": "https://www.scrumalliance.org/ScrumRedesignDEVSite/media/ScrumAllianceMedia/Files%20and%20PDFs/Infographics/S_BrandGuidelines_2018_rev.pdf"
```

Comment:
We should try to find a page where this document is linked

Reply:
It's not linked in their website at all. It only appears in web searches.